migrations/migrations/20170526115728-user-create.js

```diff
@@ -1,41 +1,47 @@
 module.exports = {
   up: (queryInterface, Sequelize) => {
-    return queryInterface.createTable('user', {
-      id: {
-        type: Sequelize.INTEGER,
-        autoIncrement: true,
-        allowNull: false,
-        primaryKey: true
+    return queryInterface.createTable(
+      'user',
+      {
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          allowNull: false,
+          primaryKey: true
+        },
+        firstName: {
+          type: Sequelize.STRING,
+          allowNull: false
+        },
+        lastName: {
+          type: Sequelize.STRING,
+          allowNull: false
+        },
+        username: {
+          type: Sequelize.STRING,
+          allowNull: false,
+          unique: true
+        },
+        email: {
+          type: Sequelize.STRING,
+          allowNull: false,
+          unique: true
+        },
+        password: {
+          type: Sequelize.STRING,
+          allowNull: false
+        },
+        created_at: Sequelize.DATE,
+        updated_at: Sequelize.DATE,
+        deleted_at: Sequelize.DATE
       },
-      firstName: {
-        type: Sequelize.STRING,
-        allowNull: false
-      },
-      lastName: {
-        type: Sequelize.STRING,
-        allowNull: false
-      },
-      username: {
-        type: Sequelize.STRING,
-        allowNull: false,
-        unique: true
-      },
-      email: {
-        type: Sequelize.STRING,
-        allowNull: false,
-        unique: true
-      },
-      password: {
-        type: Sequelize.STRING,
-        allowNull: false
-      },
-      created_at: Sequelize.DATE,
-      updated_at: Sequelize.DATE,
-      deleted_at: Sequelize.DATE
-    });
+      {
+        schema: process.env.NODE_API_DB_SCHEMA
```

Comment:
This is added automatically with any migration script config or we have to add it manually?

Reply:
manually